Helper/Cart.php

```diff
@@ -1799,4 +1843,32 @@ public function createCartByItem($item)
 
         return $cart_data;
     }
+
+    private function setQuoteIdByEncryptedUserId($quote, $encrypted_user_id)
```

Comment:
The name of this method is not well aligned with what it does. Should be better to include a variant of assignCustomer (assignQuoteCustomerBy...). And it's missing a docstring. Other than that OK.